.github/workflows/test-package.yml

```diff
@@ -64,10 +64,12 @@ jobs:
         run: dart pub get
       - name: Run VM tests
         run: dart test --platform vm
+      - name: Install coverage
+        run: dart pub global activate coverage
       - name: Collect coverage
-        run: ./tool/coverage.sh
```

Comment:
Then we should also delete that fine, not move it.

Reply:
Haha newbie mistake. Fixed it.